src/main/java/de/tum/in/www1/artemis/domain/UserGroupKey.java

```diff
@@ -0,0 +1,16 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class UserGroupKey implements Serializable {
```

Comment:
This class is needed for UserGroup, since the user_group table does not have an id.